data/config_files.yaml

```diff
@@ -23,3 +23,11 @@ sources:
   attributes: {paths: ['%%users.homedir%%/.ssh/config']}
 labels: [Configuration Files]
 supported_os: [Linux, Darwin]
+---
+name:  WordpressConfigFile
+doc: WordPress configuration file
+sources:
+- type: FILE
+  attributes: {paths: ['/**/wp-config.php']}
```

Comment:
Is there a way we could make this more specific, default wp installation locations? also know that this will only be 10 levels deep by default.

Reply:
Done, using the following three paths:

```
- '/wp/wp-config.php'
- '/var/www/wp-config.php'
- '/var/www/**/wp-config.php'
```